<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>electrical</class>
 <widget class="QWidget" name="electrical">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Electrical</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>731</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="elec_sens_layout">
    <property name="leftMargin">
     <number>300</number>
    </property>
    <item row="0" column="0">
     <widget class="QCheckBox" name="amperm_btn">
      <property name="text">
       <string>Амперметр</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="voltm_btn">
      <property name="text">
       <string>Вольтметр</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>450</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Принять</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
